katdal/datasources.py

```diff
@@ -186,9 +192,9 @@ def from_url(cls, url):
         elif url_parts.scheme == 'redis':
             # Redis server
             try:
-                telstate = katsdptelstate.TelescopeState(url_parts.netloc)
-            except redis.exceptions.TimeoutError as err:
-                raise DataSourceNotFound(str(err))
+                telstate = katsdptelstate.TelescopeState(url_parts.netloc, db)
+            except (redis.ConnectionError, redis.exceptions.TimeoutError) as e:
```

Comment:
I think both are defined in `redis.exceptions` but imported into `redis`. So you may as well be consistent and get them both from the same module.

Reply:
Done.